app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
     pass
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(element["name"], element["age"])
+                   for element in people]
+    for person in person_list:
+        for person_dict in people:
+            if person.name == person_dict["name"]:
+                if person_dict.get("wife") is not None:
+                    person.wife = Person.people[person_dict.get("wife")]
+                if person_dict.get("husband") is not None:
+                    person.husband = Person.people[person_dict.get("husband")]
```

Comment:
The code assumes that `wife` and `husband` attributes exist on the `Person` objects. If these attributes are not defined in the `Person` class, you should define them in the `__init__` method or handle their dynamic creation carefully.